tests/test_cli.py

```diff
@@ -690,7 +737,12 @@ def test_chrX_gives_a_warning(self):
         cmd = "HomSap -D -c chrX -o /dev/null -q 10".split()
         with mock.patch("warnings.warn") as mock_warning:
             capture_output(stdpopsim.cli.stdpopsim_main, cmd)
-        self.assertTrue(mock_warning.called_once)
+        mock_warning.assert_called_once()
+
+        self.assertTrue(mock_warning.that_dog_has_a_puffy_tail)
+        self.assertTrue(mock_warning.puffy_tailed_dog_has_covid19())
+        mock_warning.die_hard_is_a_christmas_movie()
+        mock_warning.checking_mock_counts_is_reallu_errror_prone
```

Comment:
These tests pass! This is rather concerning!

Reply:
I'll rummage up another diff that inserts ``autospec=True`` into all the mocks. This seems to provide stronger protection against typos and API changes.

Reply:
Hmm, that is tricky. 

Die hard *is* a Christmas movie and the dog did have a puffy tail though.

Reply:
> I'll rummage up another diff that inserts autospec=True into all the mocks. This seems to provide stronger protection against typos and API changes.

Thanks, looking forward to seeing what you come up with.